docs/whats-new/release-notes/etendo-classic/translation-bundles/procurement-extensions-es_es/release-notes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Release Notes
+tags:
+    - Release Notes
+    - Procurement
+    - Bundle
+    - Versions
+---
+:octicons-package-16: Javapackage: `com.etendoerp.procurement.extensions.es_es`
+
+:material-store: Etendo Marketplace:  [Procurement Extensions Bundle ES](https://marketplace.etendo.cloud/#/product-details?module=F863020C58E94632A7134A8031A3AA9D){target="_blank"}
+
+:octicons-project-symlink-16: Original Bundle Release Notes: [Procurement Extensions Bundle](https://marketplace.etendo.cloud/#/product-details?module=08BDBA6C314149DBA05CB1A1694F2959){target="_blank"}
+
+## Overview
+
+| Version   | Publication Date 	| From Core | To Core| Status | GitHub|
+| --- 		| ---           | ---    | ---    | --- | :---:             |
+| [1.0.0](#100) 	| 08/02/2024 	| 22.1.0 | 24.1.0 | CS	| :white_check_mark:|
+
+## Release Notes
+### 1.0.0
+- The `com.etendoerp.purchase.invoice.validations.es_es` module was added in [Procurement Extensions Bundle ES](/user-guide/etendo-classic/optional-features/bundles/procurement-extensions/overview)
```

Comment:
Actually, the translation module of the original module was added
The release date is yesterday